docs/tutorial/offscreen-rendering.md

```diff
@@ -17,22 +18,35 @@ the dirty area is passed to the `paint` event to be more efficient.
 losses with no benefits.
 * When nothing is happening on a webpage, no frames are generated.
 * An offscreen window is always created as a
-[Frameless Window](../tutorial/window-customization.md)..
+[Frameless Window](../tutorial/window-customization.md).
 
 ### Rendering Modes
 
 #### GPU accelerated
 
-GPU accelerated rendering means that the GPU is used for composition. Because of
-that, the frame has to be copied from the GPU which requires more resources,
-thus this mode is slower than the Software output device. The benefit of this
-mode is that WebGL and 3D CSS animations are supported.
+GPU accelerated rendering means that the GPU is used for composition. The benefit
+of this mode is that WebGL and 3D CSS animations are supported. There are two
+different approaches depending on whether `webPreferences.offscreen.useSharedTexture`
+is set to true.
```

Comment:
This section should explicitly call out what value of `webPreferences.offscreen.useSharedTexture` applies to which approach.  I've added a suggestion on how to do this, but feel free to word it differently.